### PR TITLE
Catch a possible crash when running  on Quilt

### PR DESCRIPTION
**Pull request by moehreag**
https://discord.com/channels/817576132726620200/832352441760088105/967656737580666881
https://discord.com/channels/817576132726620200/832352441760088105/967664157287673856

+ Added some yet missing translations

### CODE REVIEW

File: src/main/java/io/github/darkkronicle/kronhud/mixins/MixinInGameHud.java [hunk 1]
```diff
@@ -52,15 +49,19 @@ public void renderScoreboard(MatrixStack matrices, ScoreboardObjective objective
...
-        ActionBarHud hud = (ActionBarHud) KronHUD.hudManager.get(ActionBarHud.ID);
-        if (hud != null && hud.isEnabled()){
-            args.set(1, new LiteralText(""));// Set the arguments to undefined values to inhibit
-            args.set(4, 0);//                           the vanilla action bar rendering
-            hud.setActionBar(message, color);// give us selves the correct values
+                args.set(1, new LiteralText(""));// Set the arguments to undefined values to inhibit
+                args.set(4, 0);//                           the vanilla action bar rendering
+                hud.setActionBar(message, color);// give us selves the correct values
+            } catch (ClassCastException ignored) {
+                hud.setActionBar(new LiteralText("Something went wrong...(Probaly you've run this mod on Quilt)"), -1);
```
Comment (TheKodeToad) [inline src/main/java/io/github/darkkronicle/kronhud/mixins/MixinInGameHud.java:63]: Maybe the message could be:
"Something went wrong. This is usually caused by running the mod on Quilt."
Even better, you could use a translation.

**Comment by TheKodeToad** [comment]
I feel there's probably a better solution to just displaying an error.

**Comment by moehreag** [comment]
surely there is. I only don't know what could that be.. 

No, it does not occur on Fabric..

**Comment by TheKodeToad** [comment]
I think it would be better to use a @Redirect mixin.

**Comment by moehreag** [comment]
wow, thanks github

**Comment by DioEgizio** [comment]
lol

**Comment by moehreag** [comment]
Was also resolved (probably) thorugh #61

**Pull request closed without merging**